CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ Changelog
     function (@kylegulshen, gh-1043).
 -   All uses of `__future__` and `six` have been dropped (@karalekas, gh-1060).
 -   The `conftest.py` has been moved to the project root dir (@karalekas, gh-1064).
+-   In `QPU.reset()` check whether `pyquil_config.qpu_url` exists before changing
+    the endpoint (@kylegulshen, gh-1072).
```

Comment:
Move this to the bugfixes section -- "`QPU.reset()` now checks whether `pyquil_config.qpu_url` exists before updating the endpoint as not to break custom connections (@kylegulshen, gh-1072)." or something like that